src/pump/_license.py

```diff
@@ -1,6 +1,7 @@
 import os
 import logging
-from ._utils import read_json, time_method, serialize, deserialize, progress_bar, log_before_import, log_after_import
+
```

Comment:
why?

Reply:
I just added empty line, but I can remove it